src/encoded/static/components/item-pages/QualityMetricView.js

```diff
@@ -74,32 +44,49 @@ class QualityMetricViewOverview extends React.PureComponent {
         };
     }
 
+    static getSchemaQCFieldsAsOrderedPairs = memoize(function (schemaProp, context, schemas) {
+
+        if (!schemaProp && !context && !schemas) { return []; }
+
+        let propObj = null;
+        if (schemaProp) {
+            if (schemaProp.type === 'object' && schemaProp.properties) {
```

Comment:
Possible place where [JS destructuring](https://hacks.mozilla.org/2015/05/es6-in-depth-destructuring/) might help decrease amount of text/code for legibility (subjective).

Reply:
Updated.